### PR TITLE
Simulator: display additional info in logs

### PR DESCRIPTION
**Pull request by shamsimam**
Independent changes to declutter diff in cook-executor PR

### CODE REVIEW

**Comment by shamsimam** [comment]
Passing build: https://travis-ci.org/twosigma/Cook/builds/228533097

File: simulator/src/main/cook/sim/reporting.clj [hunk 1]
```diff
@@ -72,19 +72,29 @@
...
-       :turnaround (when finished-at (- finished-at action-at))
-       :overhead (overhead-time sim-job action-at instances)
-       :instance-count (count instances)}
-      (when group {:group group})))))
+      (merge
+        {:id job-id
+         :requested-at action-at
+         :name (:job/name sim-job)
+         :username (:job/username sim-job)
+         :details (into {:instances (map (fn instance-detail [instance]
```
Comment (dposada) [inline simulator/src/main/cook/sim/reporting.clj]: Should we pull `instance-detail` out into a `defn`?

Comment (pschorf) [inline simulator/src/main/cook/sim/reporting.clj]: 👍

Comment (shamsimam) [inline simulator/src/main/cook/sim/reporting.clj]: Done.

**Comment by shamsimam** [comment]
addressed feedback from @dposada  and @pschorf

**Comment by shamsimam** [comment]
Passing build: https://travis-ci.org/twosigma/Cook/builds/228758848

**Pull request merged**